parser/grammar.y

```diff
@@ -280,6 +280,22 @@ rule
       }
       tags '{' meta strings
       {
+        // Check for duplicate strings.
+        m := make(map[string]bool)
+        for _, str := range $8 {
+          ident := str.GetIdentifier()
+          // Anonymous strings (no identifiers) are fine.
+          if ident == "" {
+            continue
+          }
+          if m[ident] {
+            return asLexer(yrlex).setErrorWithLineNumber(
+              gyperror.DuplicateStringError,
+              str.GetLineNo(),
+              `rule "%s" duplicate string identifier "%s"`, $<rule>4.Identifier, ident)
```

Comment:
I think the message should be either...

`duplicate string identifier "foo" in rule "bar"` 

or 

`rule "bar": duplicate string identifier "foo"`

Without the colon after `rule "bar"` the message is read as a whole sentence that is a bit confusing.